### PR TITLE
Add Jaeger 1.8 documentation

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves https://github.com/jaegertracing/jaeger/issues/1030

* added 1.8
* renamed ElasticSearch to Elasticsearch
* removed instructions how to install - outdated and working only for macos

### CODE REVIEW

**Comment by pavolloffay** [comment]
@yurishkuro do you have architecture diagram with kafka ingester?

**Comment by pavolloffay** [comment]
cc @objectiser

File: README.md [hunk 1]
```diff
@@ -6,19 +6,9 @@ The site is built and hosted by [Netlify](https://www.netlify.com/).
...
-```
-
-This will install [Hugo](https://gohugo.io), the site's static site generator.
-
-### Linux
-
-Coming soon.
+Install the extended hugo binary from [hugo/releases]https://github.com/gohugoio/hugo/releases or
+use package manager if it is available for your operating system.
+The currently used version of hugo is defined in `netifly.toml` configuration file.
```
Comment (yurishkuro) [inline README.md]: nit: could add a link `(./netlify.toml)`

File: content/docs/1.7/deployment.md [hunk 2]
```diff
@@ -248,9 +248,9 @@ more information about choosing how many shards should be chosen for optimizatio
 Supported in Jaeger since 1.6.0
 Supported Kafka versions: 0.8+
 
-In version 1.6.0, the Kafka storage backend implementation only supports writing data, this means you will need to use another one (with the multiple storage types feature) to be able to view the traces in the Query component.
+In version 1.7.0, the Kafka storage backend implementation only supports writing data, this means you will need to use another one (with the multiple storage types feature) to be able to view the traces in the Query component.
```
Comment (yurishkuro) [inline content/docs/1.7/deployment.md]: version should not change here, I think

File: content/docs/1.7/deployment.md [hunk 3]
```diff
@@ -248,9 +248,9 @@ more information about choosing how many shards should be chosen for optimizatio
...
 
-Starting from version 1.7.0, a new component (Ingester) will be added to support reading from Kafka and storing it in another storage backend (Elasticsearch or Cassandra).
+Starting from version 1.7.0, a new component (Ingester) was be added to support reading from Kafka and storing it in another storage backend (Elasticsearch or Cassandra).
```
Comment (yurishkuro) [inline content/docs/1.7/deployment.md]: has been added

File: content/docs/1.7/deployment.md (hunk 2 above)
Comment (pavolloffay) [reply content/docs/1.7/deployment.md]: oh yes this is wrong

**Comment by pavolloffay** [comment]
The changes in 1.8 are mostly in deployment page - added ingester section

**Comment by pavolloffay** [comment]
```diff
diff -u content/docs/1.7/architecture.md content/docs/1.8/architecture.md
--- content/docs/1.7/architecture.md	2018-11-12 17:13:26.231591472 +0100
+++ content/docs/1.8/architecture.md	2018-11-12 18:38:52.581511715 +0100
@@ -48,3 +48,7 @@
 ### Query
 
 {{< definition "query" >}}
+
+### Ingester
+
+{{< definition "ingester" >}}
diff -u content/docs/1.7/deployment.md content/docs/1.8/deployment.md
--- content/docs/1.7/deployment.md	2018-11-12 20:47:26.523081469 +0100
+++ content/docs/1.8/deployment.md	2018-11-12 18:52:50.772282059 +0100
@@ -10,6 +10,7 @@
 **jaeger-agent**      | [hub.docker.com/r/jaegertracing/jaeger-agent/](https://hub.docker.com/r/jaegertracing/jaeger-agent/)
 **jaeger-collector**  | [hub.docker.com/r/jaegertracing/jaeger-collector/](https://hub.docker.com/r/jaegertracing/jaeger-collector/)
 **jaeger-query**      | [hub.docker.com/r/jaegertracing/jaeger-query/](https://hub.docker.com/r/jaegertracing/jaeger-query/)
+**jaeger-ingester**   | [hub.docker.com/r/jaegertracing/jaeger-ingester/](https://hub.docker.com/r/jaegertracing/jaeger-ingester/)
 
 There are orchestration templates for running Jaeger with:
 
@@ -26,7 +27,7 @@
 
 To see the complete list of options, run the binary with `help` command. Options that are specific to a certain storage backend are only listed if the storage type is selected. For example, to see all available options in the Collector with Cassandra storage:
 
-```
+```sh
 $ docker run --rm \
     -e SPAN_STORAGE_TYPE=cassandra \
     jaegertracing/jaeger-collector:{{< currentVersion >}} \
@@ -54,7 +55,7 @@
 
 It can be executed directly on the host or via Docker, as follows:
 
-```bash
+```sh
 ## make sure to expose only the ports you use in your deployment scenario!
 docker run \
   --rm \
@@ -73,7 +74,7 @@
 
 On Docker, a command like the following can be used:
 
-```bash
+```sh
 docker run \
   --rm \
   -p5775:5775/udp \
@@ -81,7 +82,7 @@
   -p6832:6832/udp \
   -p5778:5778/tcp \
   jaegertracing/jaeger-agent:{{< currentVersion >}} \
-  --collector.host-port=jaeger-collector.jaeger-infra.svc:14267
+  --reporter.tchannel.collector.host-port=jaeger-collector.jaeger-infra.svc:14267
 ```
 
 In the future we will support different service discovery systems to dynamically load balance
@@ -94,13 +95,13 @@
 via `--cassandra.keyspace` and `--cassandra.servers` options, or the location of Elasticsearch cluster, via
 `--es.server-urls`, depending on which storage is specified. To see all command line options run
 
-```
+```sh
 go run ./cmd/collector/main.go -h
 ```
 
 or, if you don't have the source code
 
-```
+```sh
 docker run -it --rm jaegertracing/jaeger-collector:{{< currentVersion >}} -h
 ```
 
@@ -175,7 +176,7 @@
 your Cassandra cluster correctly. After verifying with e.g. `cqlsh`, you can
 configure the collector and query like so:
 
-```
+```sh
 docker run \
   -e CASSANDRA_SERVERS=<...> \
   -e CASSANDRA_TLS=true \
@@ -248,9 +249,7 @@
 Supported in Jaeger since 1.6.0
 Supported Kafka versions: 0.8+
 
-In version 1.6.0, the Kafka storage backend implementation only supports writing data, this means you will need to use another one (with the multiple storage types feature) to be able to view the traces in the Query component.
-
-Starting from version 1.7.0, a new component (Ingester) has been added to support reading from Kafka and storing it in another storage backend (Elasticsearch or Cassandra).
+Starting from version 1.7.0, a new component [Ingester](#ingester) was added to support reading from Kafka and storing it in another storage backend (Elasticsearch or Cassandra).
 
 Writing to Kafka is particularly useful for building post-processing data pipelines.
 
@@ -278,6 +277,22 @@
 
 You can find more information about topics and partitions in general in the [official documentation](https://kafka.apache.org/documentation/#intro_topics). [This article](https://www.confluent.io/blog/how-to-choose-the-number-of-topicspartitions-in-a-kafka-cluster/) provide more details about how to choose the number of partitions.
 
+## Ingester
+**jaeger-ingester** is a service which reads span data from Kafka topic and writes it to another storage backend (Elasticsearch or Cassandra).
+
+Port  | Protocol | Function
+----- | -------  | ---
+14270 | HTTP     | Health check at **/**
+14271 | HTTP     | Metrics endpoint
+
+To view all exposed configuration options run the following command:
+```sh
+docker run \
+  -e SPAN_STORAGE_TYPE=cassandra \
+  jaegertracing/jaeger-ingester:{{< currentVersion >}} 
+  --help \
+```
+
 ## Query Service & UI
 
 **jaeger-query** serves the API endpoints and a React/Javascript UI.
diff -u content/docs/1.7/_index.md content/docs/1.8/_index.md
--- content/docs/1.7/_index.md	2018-11-12 18:37:02.925732600 +0100
+++ content/docs/1.8/_index.md	2018-11-12 18:37:02.943732564 +0100
@@ -40,6 +40,7 @@
   * Supported storage backends:
     * [Cassandra 3.4+](./deployment/#cassandra)
     * [Elasticsearch 5.x, 6.x](./deployment/#elasticsearch)
+    * [Kafka](./deployment/#kafka)
     * memory storage
 
 ## Quick Start
diff -u content/docs/1.7/monitoring.md content/docs/1.8/monitoring.md
--- content/docs/1.7/monitoring.md	2018-11-12 17:13:26.232591472 +0100
+++ content/docs/1.8/monitoring.md	2018-11-12 18:20:25.973220804 +0100
@@ -20,6 +20,7 @@
 **jaeger-agent**      | 5778
 **jaeger-collector**  | 14268
 **jaeger-query**      | 16686
+**jaeger-ingester**   | 14271
 
 ## Logging
 
```

**Comment by pavolloffay** [comment]
@objectiser agree. I will submit a follow up PR with some instructions. Thanks for the review :+1: .

**Comment by pavolloffay** [comment]
I have also thought that netifly bot will deploy the PR and post here URL. Not sure if it's broken or not configured.

File: content/docs/1.8/_index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
+title: Introduction
+weight: 1
+---
+
+Welcome to Jaeger's documentation portal! Below, you'll find information for beginners and experienced Jaeger users.
+
+If you can't find what you are looking for, or have an issue not covered here, we'd love to hear from you either on [Github](https://github.com/jaegertracing/jaeger/issues), [Gitter chat](https://gitter.im/jaegertracing/Lobby), or on our [mailing list](https://groups.google.com/forum/#!forum/jaeger-tracing).
```
[the remark this answers is not in the document]
Comment (pavolloffay) [reply content/docs/1.8/_index.md]: https://github.com/jaegertracing/documentation/issues/168

**Comment by pavolloffay** [comment]
Actually the URL with netifly deployment is in GH status indicator for this PR. So it works fine.

File: content/docs/1.8/_index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+It is used for monitoring and troubleshooting microservices-based distributed systems, including:
+
+* Distributed context propagation
+* Distributed transaction monitoring
+* Root cause analysis
+* Service dependency analysis
+* Performance / latency optimization
+
+We published a blog post, [Evolving Distributed Tracing at Uber](https://eng.uber.com/distributed-tracing/), where we explain the history and reasons for the architectural choices made in Jaeger.
+
```
[the remark this answers is not in the document]
Comment (yurishkuro) [reply content/docs/1.8/_index.md]: Sgtm to change

**Pull request merged**